index.js

```diff
@@ -11,3 +11,28 @@ mongoose
   })
   .then(x => console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`))
   .catch(err => console.error('Error connecting to mongo', err));
+
+
+const recipe = {
+  title: 'Lasagna',
+  level: 'UltraPro Chef',
+  ingredients: ['Meatloaf', 'Tomatoes'],
+  cuisine: 'Italian',
+  dishType: 'Dish',
+  duration: 80,
+  creator: 'Friso'
+}
+
+Recipe.create(recipe)
+  .then(value => console.log(`The added recipe is: ${value}`))
+
```

Comment:
Just wanted to mention to not forget adding a 'catch(err=>console.log(error)' at the end of a Promise chain.